### DISCUSSION TITLE
文档中 dither-depth 的默认值与英文手册中的不同。

### DISCUSSION THREAD

**Discussion by wathper** [hooke007/mpv_PlayKit#372] [mpv 通用]
仓库 mpv.conf 和 [中文用户手册](https://hooke007.github.io/index.html#) 中 dither-depth 一项的默认值是 no 。但是官网文档中，这项默认值是 auto 。是官网有更新吗？

**Comment by hooke007** [comment]
你如果跟着官方一起更新，就看官方的文档
